src/vs/workbench/contrib/chat/browser/agentSessions/agentSessionsActions.ts

```diff
@@ -152,18 +155,27 @@ export class AgentSessionDiffActionViewItem extends ActionViewItem {
 
 	override onClick(event: MouseEvent): void {
 		EventHelper.stop(event, true);
-
-		const session = this.action.getSession();
-
-		this.commandService.executeCommand(`agentSession.${session.providerType}.openChanges`, this.action.getSession().resource);
+		this.commandService.executeCommand(`agentSession.core.openChanges`, this.action.getSession().resource);
```

Comment:
It's preferable to call a function than do `executeCommand`. That way it's type-safe